build.gradle

```diff
@@ -191,7 +194,7 @@ uploadArchives {
             repository(url: "file://$buildDir/repos")
             if (project.hasProperty("local_archives") || ("true".equals(System.getenv("TRAVIS_PULL_REQUEST")))) {
                 repository(url: "file://$buildDir/repos")
-            } else {
+            } else if (project.hasProperty('ossrhUsername') && project.hasProperty('ossrhPassword')) {
```

Comment:
The code above uses `"` for strings, and here you are using `'`.

More globally, should these magic names be named/documented constants somewhere?

Reply:
it appears to be a mishmash, i am hesitant to fix/refactor until i can do a full clean up.